### PR TITLE
Fix: swd command message swd naming

### PR DESCRIPTION
**Pull request by perigoso**
## Detailed description

The purpose of this PR is mainly to update the SWD naming (from SW-DP to SWD ) in the SWD scan commands, since the command itself has already been renamed, it also reflects the rename of the swd scan command in the documentation, and fixes a typo and some comment farming

## Your checklist for this pull request

* [x] I've read the [Code of Conduct](https://github.com/blackmagic-debug/blackmagic/blob/main/CODE_OF_CONDUCT.md)
* [x] I've read the [guidelines for contributing](https://github.com/blackmagic-debug/blackmagic/blob/main/CONTRIBUTING.md) to this repository
* [x] It builds for hardware native (`make PROBE_HOST=native`)
* [x] It builds as BMDA (`make PROBE_HOST=hosted`)
* [x] I've tested it to the best of my ability
* [x] My commit messages provide a useful short description of what the commits do

### CODE REVIEW

File: src/command.c [hunk 1]
```diff
@@ -177,7 +177,7 @@ bool cmd_version(target_s *t, int argc, const char **argv)
 #else
 	gdb_out(BOARD_IDENT);
 	gdb_outf(", Hardware Version %d\n", platform_hwversion());
-	gdb_out("Copyright (C) 2022 Black Magic Debug Project\n");
+	gdb_out("Copyright (C) 2023 Black Magic Debug Project\n");
```
Comment (dragonmux) [inline src/command.c:180]: Particularly good catch - though this should probably state `2010-2023` or so as the project's been around way longer and one is supposed to declare a year range

File: src/command.c [hunk 2]
```diff
@@ -177,7 +177,7 @@ bool cmd_version(target_s *t, int argc, const char **argv)
 #else
```
Comment (dragonmux) [inline src/command.c:177]: The copyright and license notices should probably get copied up into the BMDA block (it needs a major rework anyway, but that's another problem)

File: src/target/adiv5_swd.c [hunk 3]
```diff
@@ -109,15 +110,18 @@ static void jtag_to_swd_sequence()
...
 	/* SWD interface must be in reset state */
 	swd_line_reset_sequence(false);
 
 	/* Send the 16-bit JTAG-to-SWD select sequence on SWDIOTMS */
 	swd_proc.seq_out(ADIV5_JTAG_TO_SWD_SELECT_SEQUENCE, 16U);
 
-	/* This ensures that if SWJ-DP was already in SWD operation before sending the select sequence, the interface enters reset state */
+	/*
+	 * This ensures that if a SWD/JTAG DP was already in SWD operation before sending the select sequence,
+	/* the interface enters reset state
```
Comment (dragonmux) [inline src/target/adiv5_swd.c:123]: the / on the front here needs removing

**Review by dragonmux — CHANGES_REQUESTED** [review]
Looking pretty good - there are a couple of tweaks it could do with but they aren't big. Let's try to land this for v1.10.

File: src/command.c (hunk 1 above)
Comment (perigoso) [reply src/command.c]: Yes, I should've gone for the proper fix instead of the quick one :) done

File: src/target/adiv5_swd.c (hunk 3 above)
Comment (perigoso) [reply src/target/adiv5_swd.c]: ups, good catch

**Review by dragonmux — APPROVED** [review]
LGTM, thank you for the quick response. Merging!

**Pull request merged**